### DISCUSSION TITLE
DataTable: Resize column not working

### DISCUSSION THREAD

**Discussion by Leshkens** [primefaces/community#2739] [PrimeVue]
Hi, changing the column width does not work when the resizable-columns property is enabled. Any ideas?
![image](https://github.com/user-attachments/assets/bac04b64-338d-4faa-b318-068a9c080008)
![123](https://github.com/user-attachments/assets/bd457ab3-a8f8-4010-b066-4606752ec0b5)

PrimeVue version: 4.0.7
Vuejs version: 3.5.8
Chrome 128

**Comment by IsaakNicolas** [comment]
Hello,

I am currently experiencing the same issue, I have also tried to reach out to the support. 
Have you managed to find a way around it ?

Wish you a good day.

**Reply by Leshkens** [reply]
Hi, vue 3.5.5 is working fine.

**Reply by IsaakNicolas** [reply]
Hello,

So I have tried on my end and it worked too !
Thank you !

Wish you a good day.
